tests/test_pretrained_models_download.py

```diff
@@ -0,0 +1,96 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+WARNING: Running this test will download ALL pre-trained NeMo models.
+This is bandwidth and disk space consuming.
+"""
+
+import nemo.collections.asr as nemo_asr
+import nemo.collections.nlp as nemo_nlp
+import nemo.collections.tts as nemo_tts
+
```

Comment:
This test file is missing a test class. But more importantly, I feel like this is a waste of testing time. I don't think we should be downloading GBs of data on every PR

Reply:
yes, this is why there is not test class. I actually just renamed it so that it isn't collected by tests. was never intended to be run with every PR

Reply:
refactored/renamed the test